packages/newspringchurchapp/src/prayer/PrayerList/index.js

```diff
@@ -1,210 +1,223 @@
-                            },
-                          ]}
-                          {...item.item}
-                        />
-                      )}
-                    </Mutation>
-                  )}
-                  ItemSeparatorComponent={() => (
-                    <DividerView>
-                      <GreyH6>Press down on card to pray</GreyH6>
-                    </DividerView>
+                >
+                  {(flagPrayer) => (
+                    <FlexedView>
+                      <ScrollArea>
+                        <ScrollView>
+                          <Header>
+                            <H6>Praying For</H6>
+                            <GreenH4>{label}</GreenH4>
+                          </Header>
+                          <StyledPrayerView>
+                            <PrayerSingleConnected
+                              avatarSize={'medium'}
+                              navigation={this.props.navigation}
+                              prayer={prayer}
+                              action={
+                                // TODO: save button component, stateful
+                                <Button>
+                                  <ChannelLabel
+                                    icon="like-solid"
+                                    label="Unsave"
+                                  />
+                                </Button>
+                              }
+                            />
+                          </StyledPrayerView>
+                        </ScrollView>
+                      </ScrollArea>
+                      <Footer>
+                        {!this.state.prayed ? (
+                          <View>
+                            <Button
+                              title={`I've prayed for ${
+                                prayer.isAnonymous
+                                  ? 'request'
+                                  : prayer.firstName
+                              }`}
+                              onPress={() => this.setState({ prayed: true })}
+                            />
+                            <FooterAltOption>
+                              <ButtonLink
+                                onPress={() =>
+                                  flagPrayer({
+                                    variables: {
+                                      parsedId: prayer.id,
+                                    },
+                                  })
+                                }
+                              >
+                                <FooterText isGray>Report Prayer</FooterText>
+                              </ButtonLink>
+                            </FooterAltOption>
+                          </View>
+                        ) : (
+                          <View>
+                            <FooterAltOption>
+                              <FooterText>
+                                Thanks for praying <Emoji name="heart" />
```

Comment:
Did we use this because we didn't already have a full heart icon?

Reply:
did this because images and text in RN don't play well together. our `Icon` component is an image.